apps/nuxt3-ssr/components/harmonization/HarmonizationListPerVariable.vue

```diff
@@ -36,7 +74,10 @@ defineProps<{
     <div>
       <ul>
         <li
-          v-for="{ cohort, status } in cohortsWithMapping.slice(20, 30)"
+          v-for="{ cohort, status } in aggregatedHarmonizationStatus.slice(
```

Comment:
I try to understand this slicing

Reply:
place holder spil into columns , can be replaced by scroll container ( the slice is not part of this PR, and older then the scroll container )